apps/wing/project-templates/wing/private-api/wing.toml

```diff
@@ -1,4 +1,11 @@
 [ tf-aws ]
+# vpc can be set to "new" or "existing"
 vpc = "new"
+# vpc_lambda will ensure that lambda functions are created within the vpc on the private subnet
 vpc_lambda = true
-vpc_api_gateway = true
\ No newline at end of file
+# vpc_api_gateway will ensure that the api gateway is created within the vpc on the private subnet
+vpc_api_gateway = true
+# The following parameters will be required if using "existing" vpc
+# vpc_id = "vpc-123xyz"
+# private_subnet_id = "subnet-123xyz"
+# public_subnet_id = "subnet-123xyz"
```

Comment:
By the way, just for general knowledge, we currently can't have two vpcs- and then put some lambdas under one of them and some under the other, right? Is it a common use case? (that someone that uses vpc tends to use more than one?)

Reply:
inter communication between VPCs is a common pattern for some architectures. This typically will require the use of either VPC peering or Transit Gateways. Id say thats probably out of scope since normally these architectures are inspired by strict organizational security concerns. Which means that the best we offer is, "you have a vpc that is peered with another? Kewl select existing and fill in the data"

As for defining multiple vpcs in the parameters and assigning different resources to each, that could be done but Id say thats a custom platform for the usecase.